src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj

```diff
@@ -17,4 +20,97 @@
     <ProjectReference Include="..\SharedTypes\SharedTypes.csproj" />
   </ItemGroup>
 
+  <Target Name="GetLinuxCrossBuildArgumentsForDNNE"
```

Comment:
Can the DNNE build moved under `src/tests`where common native infra is already wired up? We can add a simple cmake file for DNNE project there (as we have for other external projects  libunwind, brotli etc.), without replicating the script code so implicitly.

Reply:
We'd rather not move the DNNE project stuff over to the `src/tests` tree as that will prevent us from using XUnit for our tests easily. We're also trying to keep the DNNE native component very slim and as transparent to the build as possible, and moving it into the CMake build for tests would make it not transparent due to build dependencies (the native C file is generated from the compiled managed assembly)